src/Components/PetList.js

```diff
@@ -1,9 +1,20 @@
-export const PetList = () => {
-  return (
-    <aside className="pets-list">
-      <p>No pets listed for this employee.</p>
-    </aside>
-  );
-};
+// import { useState } from "react";
+
+
+
+export const PetList = ({petid, docid, petname,index}) => {
```

Comment:
I would start with the assumption that the PetList is meant to render all the pet names, rather than one by one. The most important thing with breaking down a project is to reduce the amount of work we have to deal with. Having a separate component per pet name would be a major headache to deal with